db/seeds.rb

```
@@ -5,3 +5,23 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+Image.create(url: 'http://sfwallpaper.com/images/pretty-nature-wallpaper-1.jpg')
```

we should raise an exception if creating one of these fails

We can also pass an array of hashes to a single call to `Image.create!`